### ISSUE TITLE
1.19.2: Can't enter my world

### ISSUE THREAD

**Issue by SeverusSnow** [Reascer/WeaponsOfMinecraft-1.18.2#55]
Epic Fight Version: 19.5.17
Weapons of Miracles Version: 19.1.7.34
No Datapacks

So previous Crash-Bug has been fixed but now if I try to join my world it gives me some sort of Error-Message and when I click on "Safe Mode" it just doesn't do anything. I tried to delete the datapack folder but that didn't help either.

Here is a picture of how it looks:
![Error](https://github.com/Reascer/WeaponsOfMinecraft-1.18.2/assets/98117985/fe9b3d13-b279-4d18-8811-1bb8c008de21)

**Comment by Reascer** [comment]
can you send me the log ?

**Comment by SeverusSnow** [comment]
> can you send me the log ?

Since it didn't crash I didn't get a crash-log, but for some reason I can join my world now. I don't know what I did but it seems to work fine now, I joined my world and everything was in place. But this window appeared yesterday everytime I tried to open my world and when I clicked on "Safe Mode" it didn't do anything but now it works so the problem might got fixed when PC restarted.
